src/main/java/base/HelloWorld.java

```diff
@@ -0,0 +1,43 @@
+package base;
+
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String str;
+    private int var;
```

Comment:
`var` is a bad variable name because from Java 11 this word is used to create a variable without variable type definitions